src/CONST.js

```diff
@@ -423,6 +423,10 @@ const CONST = {
         ERROR: {
             USER_CANCELLED: 'User canceled flow',
             USER_TAPPED_BACK: 'User exited by clicking the back button.',
+            USER_CAMERA_DENINED: 'Onfido.OnfidoFlowError',
+            USER_CAMERA_PERMISSION: 'Encountered an error: cameraPermission',
+            // eslint-disable-next-line max-len
+            USER_CAMERA_CONSENT_DENIED: 'Unexpected result Intent. It might be a result of incorrect integration, make sure you only pass Onfido intent to handleActivityResult. It might be due to unpredictable crash or error. Please report the problem to android-sdk@onfido.com. Intent: null \n resultCode: 0',
```

Comment:
Let's wrap this in a new line and remove the Eslint comment.

Reply:
There are some reasons that I would like to keep it as it is.
- Linting required single quote, if we want to wrap the message we need to use backquote **``** it will cause an error by lint rule. However, the condition to match this message variable should be 100% match when the error cause, if we wrap it into 2 lines, the condition couldn't work as expected.
- From my point of view, this is a message under constant variable and not a block logic code so it should not a big deal.

Reply:
Ok. Anyways, it is not a blocker.